frontend/public/components/nav/items.tsx

```diff
@@ -104,6 +105,7 @@ export type NavLinkProps = {
   disallowed?: string;
   startsWith?: string[];
   activePath?: string;
+  testId?: string;
```

Comment:
Per console naming conventions, this should be `testID`

```suggestion
  testID?: string;
```

Reply:
I've raised a separate PR because this change will be used by other people to incorporate their changes with respect to the changes in the PR.

> Per console naming conventions, this should be `testID`

I'll change the PR

Reply:
> I've raised a separate PR because this change will be used by other people to incorporate their changes with respect to the changes in the PR.

But this change should be in those individual PRs. It doesn't make sense to add a test-id that isn't being used yet by a test. The test-id may or may not be needed based on feedback in the specific test PR.

Reply:
Alright I'll include these changes with the test PR thanks